policyengine_us/parameters/gov/states/nc/ncdhhs/scca/disabled_age_limit.yaml

```diff
@@ -0,0 +1,11 @@
+description: North Carolina limits its Subsidized Child Care Assistance program to disabled children below this age.
+metadata:
+  reference:
+    - title: Child Care and Development Fund (CCDF) Plan for State/Territory North Carolina FFY 2022 – 24 Amendment 1
+      href: https://ncchildcare.ncdhhs.gov/Portals/0/documents/pdf/A/ACF-118_CCDF_FFY_2022-2024_For_North_Carolina_Amendment_1.pdf?ver=C9YfIUPAFekeBA3I1mN8aA%3d%3d#page=83
+  label: North Carolina Subsidized Child Care Assistance program disabled child age limit
+  unit: year
+  period: year
+# Confirmed with our benefits specialist: the disabled age limit is exclusive to 18.
```

Comment:
as in 18 year olds are ineligible? the document says under 19

Reply:
I've doubled confirmed with our specialist, 18 exclusive. Also the document mentions 17. 
<img width="200" alt="Screenshot 2025-02-21 at 9 55 47 AM" src="https://github.com/user-attachments/assets/04a3a42c-9985-45d7-b38e-d9642acc8205" />

Reply:
ah i misread. if the document says 17 please change it to 17 and make the logic <=

Reply:
I think "< 18" is better. A 17-and-a-half-year-old would still be eligible, but changing it to "<= 17" would make them ineligible. Am I overthinking this?